app/models/classification.rb

```diff
@@ -521,6 +524,7 @@ def self.tag2human(tag)
 
     "#{cname}: #{ename}"
   end
+  private_class_method :tag2human
```

Comment:
good find. Guess we need a test for `Classification.tag2human` (not part of this PR)